mod_reforged/hooks/skills/actives/adrenaline_skill.nut

```diff
@@ -2,6 +2,21 @@
 	q.create = @(__original) function()
 	{
 		__original();
+		this.m.FatigueCost = 10;	// In vanilla this is 20
```

Comment:
This change should also be mentioned in the commit message. An example could be: `refactor: buff adrenaline by reducing fatigue cost and ignoring injuries` that fits within the character limit for commit messages.